apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js

```diff
@@ -59,23 +54,13 @@ $(document).ready(() => {
   function init() {
     let hiddenUserType = $('#user_user_type').attr('type') === 'hidden';
     if (!hiddenUserType) {
-      if (inClearerUserTypeRollout) {
-        // If in variant, toggle large buttons
-        document.getElementById('select-user-type-original').style.cssText =
-          'display:none;';
-        document.getElementById('select-user-type-variant').style.cssText =
-          'display:flex;';
-        document.getElementById('signup-select-user-type-label').style.cssText =
-          'width:135px;';
-      } else {
-        // Otherwise (also the default), keep original dropdown
-        document.getElementById('select-user-type-variant').style.cssText =
-          'display:none;';
-        document.getElementById('select-user-type-original').style.cssText =
-          'display:flex;';
-        document.getElementById('signup-select-user-type-label').style.cssText =
-          'width:220px;';
-      }
+      // If in variant, toggle large buttons
```

Comment:
Nit: I think this comment can be deleted too!

Reply:
Ahh good catch! Updated!